cli/util/path.rs

```diff
@@ -405,6 +377,34 @@ mod test {
     }
   }
 
+  /// Gets a path with the specified file stem suffix.
+  ///
+  /// Ex. `file.ts` with suffix `_2` returns `file_2.ts`
+  pub fn path_with_stem_suffix(path: &Path, suffix: &str) -> PathBuf {
```

Comment:
This can go. It is only used in `test_path_with_stem_suffix`, which can also go.